common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -173,7 +173,7 @@ class BulkOperationsMixin(object):
     mongo_connection.
     """
     def __init__(self, *args, **kwargs):
-        super(BulkOperationsMixin, self).__init__(*args, **kwargs)
+        super(BulkOperationsMixin, self).__init__()
```

Comment:
@blarghmatey I see what you're saying, in which case we should just remove the line completely, there's no point invoking `__init__()` on `object`, is there?

@nedbat It seems like we're allowing `*args` and `**kwargs` because we don't know how `BulkOperationsMixin` will be subclassed, so yes, multiple (and nested) inheritance.